src/proc.cpp

```diff
@@ -453,40 +453,34 @@ static void process_mark_finished_children(parser_t &parser, bool block_ok) {
     reap_disowned_pids();
 }
 
-/// Given a command like "cat file", truncate it to a reasonable length.
-static wcstring truncate_command(const wcstring &cmd) {
-    const size_t max_len = 32;
-    if (cmd.size() <= max_len) {
-        // No truncation necessary.
-        return cmd;
+/// Call the fish_job_summary function with the given args.
+static void print_job_summary(parser_t &parser, wcstring_list_t args) {
```

Comment:
`args` should have type `const wcstring_list_t &args` to avoid a copy

Reply:
Is it that easy to tell I don't use C++ elsewhere? :)
Done.